icons/story.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path
```

Comment:
The paths should not be a compound path (combined path). Design programs like Figma don't like them. Please split them up in separate elements/

Reply:
You can just paste it in https://lucide-studio.vercel.app and press <kbd>Tidy</kbd>.

Reply:
Ahh will do! I've been using SVGOMG as per documentation which combines all the paths, even though I agree that keeping them separate makes it easier to make changes later. That link would be great in the docs @jguddas

Reply:
I split up the paths and also made some minor adjustments to have an almost exact 2px spacing between dashes (hard with ellipses).